acceptance-tests/src/test/java/tech/pegasys/signers/dsl/hashicorp/HashicorpVaultCommands.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.signers.dsl.hashicorp;
+
+import java.util.Map;
+
+// This assumes there is a Vault Server running at teh vault URL (for AT, typically in a docker)
+public class HashicorpVaultCommands {
+
+  private final String vaultUrl;
+
+  public HashicorpVaultCommands(final String vaultUrl) {
+    this.vaultUrl = vaultUrl;
+  }
+
+  public String[] vaultStatusCommand() {
```

Comment:
nit: no need to prefix all these commands with vault

Reply:
true